src/js/components/sharedComponents/checkbox/PrimaryCheckboxType.jsx

```diff
@@ -111,18 +111,16 @@ export default class PrimaryCheckboxType extends React.Component {
     }
 
     toggleChildren() {
-        let showChildren = true;
-        let arrowState = 'expanded';
+        // TO-DO: Fix input for bulkTypeChange on award type filter
```

Comment:
1. todos require your name on them so we know who wrote it in the future
2. I don't understand this bug, please explain it further